docs-source/content/userguide/managing-domains/domain-lifecycle/introspection.md

```diff
@@ -44,6 +44,21 @@ Set `introspectVersion` to a new value.
 
 As with `restartVersion`, the `introspectVersion` field has no required format; however, we recommend using a value likely to be unique such as a continually increasing number or a timestamp.
 
+Beginning with operator 3.1.0, if a domain resource's `spec.introspectVersion` is set, each of the domain's WebLogic Server pods will have a label with the key `weblogic.IntrospectVersion` to indicate the `introspectVersion` at which the pod is running.
+
+```
+Name:           domain1-admin-server
+Namespace:      domain1-ns
+Priority:       0
+Labels:         weblogic.createdByOperator=true
+                weblogic.domainName=domain1
+                weblogic.domainUID=domain1
+                weblogic.introspectVersion=12345
+                weblogic.serverName=admin-server
```

Comment:
restartVersion?

Reply:
It is just an example to show the new label. startVersion is not always set.

Reply:
Up to you.  introspectVersion is not always set either.

Reply:
I am sure there are many things that are in the pod but not in this example. We are talking about the use case where the  introspect version is set here.